renku_notebooks/api/schemas.py

```diff
@@ -77,14 +77,21 @@ def unnest_keys(self, data, **kwargs):
         return flatten_dict(data)
 
 
-class UserPodResources(Schema):
+class UserPodResources(
```

Comment:
I think this is equivalent to simply doing
```
    UserPodResources = Schema.from_dict(
        {
            "cpu": fields.Str(required=True),
            "memory": fields.Str(required=True),
            "ephemeral-storage": fields.Str(required=False),
        }
    )
```

Reply:
Yeah I think you are right and it looks way better. I will change it.